src/main/java/org/openstreetmap/atlas/checks/distributed/IntegrityCheckSparkJob.java

```diff
@@ -96,8 +95,8 @@
             "Saves intermediate atlas files created when processing OSM protobuf data.",
             Boolean::valueOf, Optionality.OPTIONAL, "false");
     private static final Switch<Set<OutputFormats>> OUTPUT_FORMATS = new Switch<>("outputFormats",
-            String.format(
-                    "Comma-separated list of output formats (flags, metrics, geojson, tippecanoe)."),
+            String.format("Comma-separated list of output formats %s and %s and %s and %s ",
+                    "flags", "metrics", "geoJson", "tippeCanoe"),
```

Comment:
I am guessing spectrum was complaining about using String.format without only one argument. All of these arguments are just string literals, so we can just use the original string without calling String.format.